features/enable_fips_cloud.feature

```diff
@@ -135,7 +135,7 @@ Feature: FIPS enablement in cloud based machines
             Updating <fips-name> package lists
             Installing <fips-name> packages
             <fips-name> enabled
-            A reboot is required to complete install
+            A reboot is required to complete install.
```

Comment:
very pedantically speaking, this is a regexp so `.` could be anything
Of course we expect a dot and expect it to pass like this but `\.` would be more accurate (:

Reply:
This was discussed in person at stand-up today, and the conclusion was that changing these checks to "contains substring" was the most appropriate solution.